R/mod_review_forms.R

```diff
@@ -231,8 +226,7 @@ mod_review_forms_server <- function(
       review_save_error(FALSE)
       golem::cat_dev("Save review status reviewed:", input$form_reviewed, "\n")
       
-      review_row <- review_data_active() |> 
-        dplyr::distinct(subject_id, item_group) |> 
+      review_records <- review_data_active() |> 
```

Comment:
We could still only select the id column so that review_records only contains the minimum necessary information for the update. Something like below

```
review_records <- review_data_active()["id"] |>
    ...
```